compiler-rt/lib/scudo/standalone/primary32.h

```diff
@@ -187,6 +191,20 @@ template <typename Config> class SizeClassAllocator32 {
     return BlockSize > PageSize;
   }
 
+  u16 popBlocks(CacheT *C, uptr ClassId, CompactPtrT *ToArray) {
+    TransferBatch *B = popBatch(C, ClassId);
+    if (!B)
+      return 0;
+
+    const u16 Count = B->getCount();
```

Comment:
There used to be a DCHECK that count was not zero. Should that DCHECK still be here?

Reply:
Nice catch!